### PR TITLE
Fix up typos

### PR DESCRIPTION
**Pull request by mimischi**
Changes made in this Pull Request:
- Fix `import` artifact in headers of the python files.
- Update `README.rst`

PR Checklist
------------
 - ~~CHANGELOG updated?~~

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,21 +5,23 @@
...
 
 
 Installation
 ============
 
-The package is currently under development. Refer to the `installation wiki page <https://gitlab.mpcdf.mpg.de/MPIBP-Hummer/MDBenchmark/wikis/installation>`_ for instructions.
+The package is currently under development.
 
 .. code::
     python setup.py install --user
```
Comment (kain88-de) [inline README.rst]: needs an empty line

**Comment by kain88-de** [comment]
while you are cleaning up the readme. You can remove the line about mdsynthesis as we will remove it in the future.

File: README.rst (hunk 1 above)
Comment (mimischi) [reply README.rst]: Where does the empty line go?

Comment (mimischi) [reply README.rst]: Nevermind.

File: README.rst [hunk 2]
```diff
@@ -112,13 +113,3 @@ directory.
...
-
-Notes
-=====
-
-The MDBenchmark tool uses `mdsynthesis`_ to generate the folder sub structure for
-each MDBenchmark run. You can also use `mdsynthesis`_ in a python process later to
-analyze the MDBenchmarks and check your simulations.
-
-.. _mdsynthesis: https://mdsynthesis.readthedocs.io/en/master/
-.. _xdg: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
```
Comment (kain88-de) [inline README.rst]: this reference could stay ;)

Comment (mimischi) [reply README.rst]: Right. Fixed.

**Comment by kain88-de** [comment]
thanks

**Pull request merged**